README.md

```diff
@@ -96,8 +96,45 @@ Libraries are built by running SBT's built-in `publishLocal` task and then copyi
 
 ### Source Dependencies
 
-Sbtix builds can depend on other Sbtix builds by adding the attr `sbtixBuildInputs` to their call to `buildSbt*`. It's used like Nix's `buildInputs`,
-but makes them available to Ivy/Coursier.
+Sbtix builds can depend on dependencies not found in online repositories by adding the attr `sbtixBuildInputs` to their call to `buildSbt*`.
+
+To make sure transitive dependencies and overrides work correctly,
+the local dependency must be available both when running 'sbtix-gen'
+and when building. You provide it in a `sbtix-build-inputs.nix` file,
+which could hold a single dependency:
+
+```
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.callPackage ./scala-scraper {}
```

Comment:
We need to make it clear in README that `scala-scraper` is the local dependency. It seems you assume that it already contains Nix files generated by Sbtix. If so, can you write a workflow? Something like:
1. do step 1 in the local dependency `scala-scraper`
2. do step 2 in the current project
3. ...

Reply:
I've expanded on this a bit, could you have a look if this makes it clearer?